config/backends.go

```diff
@@ -45,15 +45,16 @@ const (
 )
 
 type Aerospike struct {
-	DefaultTTL      int      `mapstructure:"default_ttl_seconds"`
-	Host            string   `mapstructure:"host"`
-	Hosts           []string `mapstructure:"hosts"`
-	Port            int      `mapstructure:"port"`
-	Namespace       string   `mapstructure:"namespace"`
-	User            string   `mapstructure:"user"`
-	Password        string   `mapstructure:"password"`
-	MaxReadRetries  int      `mapstructure:"max_read_retries"`
-	MaxWriteRetries int      `mapstructure:"max_write_retries"`
+	DefaultTTL            int      `mapstructure:"default_ttl_seconds"`
+	Host                  string   `mapstructure:"host"`
+	Hosts                 []string `mapstructure:"hosts"`
+	Port                  int      `mapstructure:"port"`
+	Namespace             string   `mapstructure:"namespace"`
+	User                  string   `mapstructure:"user"`
+	Password              string   `mapstructure:"password"`
+	MaxReadRetries        int      `mapstructure:"max_read_retries"`
+	MaxWriteRetries       int      `mapstructure:"max_write_retries"`
+	ConnectionIdleTimeout int      `mapstructure:"connection_idle_timeout"`
```

Comment:
@guscarreon Can we rename this to be `connection_idle_timeout_sec` to be clear about the time unit? Also, can you please add a comment for this config filed along the lines of:
```
// Please set this to a value lower than the `proto-fd-idle-ms` (converted to seconds) value set in your Aerospike Server. 
// This is to avoid having race conditions where the server closes the connection but the client still tries to use it.
// If set to a value less than or equal to 0, Aerospike Client's default value will be used which is 55 seconds.
```

Reply:
Makes sense. Great comment wording. Modified